SpatialGDK/Source/SpatialGDK/SpatialGDK.Build.cs

```diff
@@ -18,6 +18,11 @@ public SpatialGDK(ReadOnlyTargetRules Target) : base(Target)
 
         PrivateIncludePaths.Add("SpatialGDK/Private");
 
+        var WorkerSDKPath = Path.GetFullPath(Path.Combine(ModuleDirectory, "Public", "WorkerSDK"));
+
+        PublicIncludePaths.Add(WorkerSDKPath); // Worker SDK uses a different include format <improbable/x.h>
+        PrivateIncludePaths.Add(WorkerSDKPath);
```

Comment:
What's the include that requires these path changes? If the header is only used internally, we don't need to add it to the public include paths.

Reply:
I think some files end up including workerSDK headers from the game project, perhaps unnecessarily so i'll investigate that

Reply:
This needs quite a bit of work to get working, I have instead created a task for it here https://improbableio.atlassian.net/browse/UNR-2644

Reply:
I'm a bit unsure about this change, come chat with me :D